.github/workflows/tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on:
+  push:
+    branches:    
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+        id: go
+
+      - name: Get dependencies
+        run: |
+          go get -u gotest.tools/gotestsum
+          go get google.golang.org/grpc/internal/transport@v1.41.0
+          go get google.golang.org/grpc@v1.41.0
```

Comment:
I do not like this, but this is the only way I've found to discover those modules: I added them locally, but after running `go mod tidy` they are pruned from the modules, causing the Go process to not discover them. 🤷